aten/src/ATen/native/sparse/SparseTensorMath.cpp

```diff
@@ -275,6 +275,27 @@ Tensor norm_sparse(const SparseTensor& self, Scalar value) {
   return self.coalesce()._values().norm(value);
 }
 
+// --------------------------------------------------------------------
+// mv(SparseTensor, Tensor)
+// --------------------------------------------------------------------
+
+Tensor mv_sparse(const SparseTensor& self, const Tensor& vec)
+{
+  AT_ASSERT(self.is_sparse());
```

Comment:
I think an is_sparse() call will always be true for a SparseTensor type.

Reply:
I remove this check.